app/pb_kits/playbook/pb_user/user.rb

```diff
@@ -40,6 +41,13 @@ def avatar_size
       def title_size
         size == "lg" ? 3 : 4
       end
+
+      def show_territory
```

Comment:
I would consider making this a little more declarative and push more of the logic into this object and pull it out of the view. Maybe name this details or territory_and_title.

``` ruby
[territory, title].compact.join(“•”)
```

I would also consider using a unordered list and css to show the “•”.

Reply:
What are the benefit(s) of using unordered list vs. just using " • "? @terryfinn

Reply:
You avoid the delimiter from being text nodes like the other values. That means they will not be copyable, and they will not be read by a screen reader.